CMakeLists.txt

```diff
@@ -146,7 +122,7 @@ if(CAMODOCAL_PLATFORM_UNIX)
     #set_linker_flags("-Wl,--no-undefined -ldl -lnsl -lm -pthread -lrt" CACHE)
     find_package(LibDL)
     find_package(Libnsl)
-    find_package(LibM)
+    # find_package(LibM)
```

Comment:
I believe that libM change will produce problems on other platforms. What happens when find_package(Libm) is used instead? that should fix the apparent capitalization change issue